README.md

```diff
@@ -102,6 +102,12 @@ This runs steps 1-5 in the -WhatIf output from above. You can now run dotnet Mus
 
 ## Other things you can do
 
+### Use AllReady app
+
+MusicStore is the default app being executed with `RunBenchmark.ps1`. If you want to try the `AllReady` app you need to provide two extra arguments to the script:
+
+`-WebAppFolder src\AllReady -WebAppDllName AllReady.dll`
```

Comment:
I suggest making these command lines as simple as possible. For example add just one arg that is focused on intent and make it the script's responsibility to know what source folder / dll to use.

-test [AllReady|MusicStore]

If not specified MusicStore remains the default.